src/puya/puya-service.ts

```diff
@@ -0,0 +1,166 @@
+import { spawn } from 'child_process'
+import * as rpc from 'vscode-jsonrpc/node'
```

Comment:
@neilcampbell i assume extensions will require node or are we trying to target web environments?